docs/designs/helmify.md

```diff
@@ -0,0 +1,44 @@
+# Helm-ify
+
+## The problem
+
+Currently the project relies on helm install-ing several helm Charts,
+required for the various components. 
+However, other parts of the project such as the project components themselves 
+and migrations are a combination of `kustomize.yaml` manifests (components), k8s
+ yamls applied through `draconctl` (migrations). This is significantly confusing
+and tech debt since it is not clear what installs what, how and why.
+
+It is suggested that the project uses Helm as its sole way of deploying things.
+
+## Changes suggested
+
+The following changes are suggested
+
+* Make a `dracon-dev` chart which installs all the sub-charts referenced by 
+the `make-dev` target
+* Make `draconctl migrations apply` only apply migrations against a database url
+and not try to manage a k8s job
+* Make a `dracon` chart which installs dracon components and migrations only
+assuming there are prerequisite systems present (Elasticsearch, Postgres etc),
+values for those systems are provided through `values.yaml`
+* Transform all components into Charts, one chart for each.
```

Comment:
We have discussed this offline that it's not necessary at this stage, we might revisit it in the future and even if it was, it's not part of the work this RFC is supposed to tackle